providers/github/resources/github_repo.go

```diff
@@ -584,7 +584,7 @@ func newMqlGithubCommit(runtime *plugin.Runtime, rc *github.RepositoryCommit, ow
 		}
 	}
 
-	if rc.Author != nil {
+	if rc.Author != nil && rc.Author.ID != nil && rc.Author.Login != nil {
```

Comment:
what are the panics that require those changes?
`llx.IntDataPtr` should have a nil check and fall back to returning the MQL equivalent of nil

Reply:
these things get passed to the init function for user and that function then crashes, we shouldn't be trying to create a user if such didn't exist for that commit... this is just github's weird way of saying there's no author itseems... the author field is set but all props are `nil`